app/SharedExpense.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SharedExpense extends Model
```

Comment:
yeah we can use json_decode and json_encode. But basically this lets us only have one row in the expense table for an expense but with a json field, we can tie as many users as we want to the expense.  Saves us from having a long association table.